project/.php_cs.dist

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * DO NOT EDIT THIS FILE!
+ *
+ * It's auto-generated by sonata-project/dev-kit package.
+ */
+
+$header = <<<'HEADER'
+This file is part of the Sonata Project package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+HEADER;
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
+    ->exclude('Tests/Fixtures')
+    ->exclude('tests/Fixtures')
+    ->exclude('Resources/skeleton')
+    ->exclude('Resources/public/vendor')
+;
+
+return PhpCsFixer\Config::create()
+    ->setFinder($finder)
+    ->setRules([
+        '@Symfony' => true,
```

Comment:
Could you please add the following rules set?

- Symfony:risky
- PHP56Migration
- PHP56Migration:risky
- PHPUnit57Migration:risky